dask_jobqueue/tests/test_htcondor.py

```diff
@@ -42,6 +42,8 @@ def test_job_script():
         assert 'MY.JobId = "$(ClusterId).$(ProcId)"' in job_script
         assert "LANG=en_US.utf8" in job_script
         assert "LC_ALL=en_US.utf8" in job_script
+        assert "cd /some/path/" in job_script
+        assert "source venv/bin/activate" in job_script
         assert "export" not in job_script
```

Comment:
a) Why is export stripped from jobscript ?? Why doesn't this fail?
b) Why can't this be checked independent og htcondor in the generic part

Reply:
For b), yes this could now be checked independently of HTCondor, we could add another test for `env_extra` in test_jobqueue_core.py, and apply it with a fixture to all the cluster classes.

But since the behavior just changed for HTCondor, I'm keen to leave also this part of the assertion in the HTCondor cluster.

> Sadly there seem to be problems with the CI in general, right?

Yes unfortunately, I'm trying to advance on this point, but I've not a lot of time to devote to it.

Reply:
Alright, I realized anyway with the addition of f678d37b709eb24e927b9a250ac5aa9d1fca7de0 that HTCondor will stay special because of the quoting. At least for the environment variable part (see that commit) you can't generalize this to all Cluster classes.

Reply:
Oh yeah, special quoting problem. So this confirm we need to keep some specific test for HTCondor. 

If you feel like adding a more general test for every queueing system (without export statement 😀), please do.